src/main/java/application/service/BookServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package application.service;
+
+import application.dto.CreateBookRequestDto;
+import application.exception.EntityNotFoundException;
+import application.mapper.BookMapper;
+import application.model.Book;
+import application.repository.BookRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class BookServiceImpl implements BookService {
+    private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
+
+    @Override
+    public Book save(Book book) {
+        return bookRepository.save(book);
+    }
+
+    @Override
+    public List<Book> findAll() {
+        return bookRepository.findAll();
+    }
+
+    @Override
+    public Book createBook(CreateBookRequestDto requestDto) {
+        return bookRepository.save(bookMapper.toModel(requestDto));
+    }
+
+    @Override
+    public Book getBookById(Long id) {
+        Book book = bookRepository.findById(id)

```

Comment:
Avoid declaring redundant variable. You can directly return the result of method call.
Think about it across all project.